### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by dubeyaditya29**
Added Wishlist page
Screenshot (81)

### CODE REVIEW

File: markups/login.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        <a href="#" class="logo_link">
+          <i class="fas fa-birthday-cake fa-lg cake-logo"></i>
+        </a>
+        <h1 class="heading-title">Cmart</h1>
+      </div>
+      <ul class="list">
+        <!-- <li class="list_item">
+          <input type="text" class="search_navbar" placeholder="Search..."></input>
+          <i class="fas fa-search"></i>
+        </li> -->
```
Comment (Hamdrive) [inline markups/login.html:26]: cleanup commented code

File: markups/signup.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        <a href="#" class="logo_link">
+          <i class="fas fa-birthday-cake fa-lg cake-logo"></i>
+        </a>
+        <h1 class="heading-title">Cmart</h1>
+      </div>
+      <ul class="list">
+        <!-- <li class="list_item">
+          <input type="text" class="search_navbar" placeholder="Search..."></input>
+          <i class="fas fa-search"></i>
+        </li> -->
```
Comment (Hamdrive) [inline markups/signup.html:26]: commented code

**Review by Hamdrive — COMMENTED** [review]
Good work done, have suggested to remove the commented code

File: markups/login.html (hunk 1 above)
Comment (dubeyaditya29) [reply markups/login.html:26]: Have removed it.

File: markups/cart.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    <link
+      rel="stylesheet"
+      href="https://aceeui.netlify.app/stylesheet/final.style.css"
+    />
+    <link rel="stylesheet" href="/styles/cart.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="logo">
+        <a href="#" class="logo_link">
```
Comment (kedark152) [inline markups/cart.html:17]: You can add home-page link in logo.

File: markups/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    <link
+      rel="stylesheet"
+      href="https://aceeui.netlify.app/stylesheet/final.style.css"
+    />
+    <link rel="stylesheet" href="/styles/cart.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="logo">
+        <a href="#" class="logo_link">
```
Comment (kedark152) [inline markups/wishlist.html]: You can add home-page link in logo.

File: scripts/login.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+const content = document.querySelector(".input input");
```
Comment (kedark152) [inline scripts/login.js]: you can give different class-name to `.input`

File: styles/cart.css [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+  position: absolute;
+  top: 43%;
+  left: 26%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.list_end {
+  /* padding-left: 10%; */
```
Comment (kedark152) [inline styles/cart.css]: you can remove commented code, if not required in future

File: styles/wishlist.css [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+  left: 26%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.list_end {
+  /* padding-left: 10%; */
+}
+.footer {
```
Comment (kedark152) [inline styles/wishlist.css:21]: If you are using same footer code in multiple files, then simply create one single `footer.css` and import it in each file.

File: styles/signup.css [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+  left: 21%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.list_end {
+  padding-left: 10%;
+}
+.footer {
```
Comment (kedark152) [inline styles/signup.css:21]: If you are using same footer code in multiple files, then simply create one single `footer.css` and import it in each file.

**Review by kedark152 — APPROVED** [review]
Code written very well,
If time persists then, just try to make suggested changes like creating `footer.css` file and importing wherever required rather than copying same code in every file.
also we need to merge all our PR to `dev` named branch, please check if you can re-name `main` branch to `dev`

File: scripts/login.js (hunk 5 above)
Comment (dubeyaditya29) [reply scripts/login.js]: made the required changes

File: styles/wishlist.css (hunk 7 above)
Comment (dubeyaditya29) [reply styles/wishlist.css:21]: Nice thing you pointed out thanks

**Comment by dubeyaditya29** [comment]
Thanks @kedark152 for great suggestions for footer.css if time permits I will make those changes rest I have made the changes and will  be merging now.

**Pull request merged**